Makefile

```diff
@@ -194,6 +194,26 @@ config_morse_localnet: ## Create a localnet config file to serve as a Morse gate
 		echo "##################################################################################################################"; \
 	fi
 
```

Comment:
I looked at the top of this Makefile and it seems like it's still using `docker compose`. 

Please merge & consolidate with `main`.

Reply:
I pulled main and verified the changes in main. The docker-compose section is still [there](https://github.com/buildwithgrove/path/blob/main/Makefile#L14-L28). I will remove those targets as well as the tilt ones in favor of the configuration I am pushing in this PR.